patches/0001-makefile-Add-install-target-and-build-the-module-by-.patch

```diff
@@ -27,7 +27,7 @@ index 44c7bb83981a..2fee5e096b51 100755
  clean:
  	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) clean
  
-+install: modules
++install:
 +	$(MAKE) ARCH=$(ARCH) -C $(KDIR) M=$(CURDIR) INSTALL_MOD_PATH=$(INSTALL_MOD_PATH) modules_install
 +
```

Comment:
I'd prefer to properly address the issue. If you just run make install now it will fail because the artifacts have not been built.

Reply:
Yes you're right, that's the point.
Do I need to re-create pull request with corrected log?